benchexec/tools/metaval++.py

```diff
@@ -45,22 +44,33 @@ def cmdline(self, executable, options, task, rlimits):
 
         return [executable] + options + list(task.single_input_file)
 
+    @staticmethod
+    def substring_after_identifier(string, identifier, occurrences=1):
+        return (
+            identifier.join(string.split(identifier)[occurrences:])
+            if occurrences > 0
+            else string
+        )
```

Comment:
Are you just looking for the `maxsplit` parameter of `str.split()`?

Why define an optional parameter that you are never using?

Reply:
Thanks a lot for your comments!

`maxsplit` was exactly what I was looking for